Commands.md

```diff
@@ -1,95 +0,0 @@
-## Commands
```

Comment:
We should update README to show the new way

Reply:
Oops, good point. Going to push another commit for that.